packages/api/utils/ipLogging.ts

```diff
@@ -0,0 +1,14 @@
+import middy from "@middy/core";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+
+// Create the middleware and export it
+export const IpCheckerMiddleware = (): middy.MiddlewareObj<APIGatewayProxyEvent, APIGatewayProxyResult> => {
+  // Set up a before check, this will run before the handler
+  const before: middy.MiddlewareFn<APIGatewayProxyEvent, APIGatewayProxyResult> = async (request): Promise<void> => {
+    // log the sourceIp
+    console.log(request.event.requestContext.identity.sourceIp);
```

Comment:
In addition to added context per Jeff, I suggest using the `info`-level to log.

This would necessitate a corresponding change to any unit test with a spy method for _log_.
`const consoleLogSpy = jest.spyOn(console, "log");`

```suggestion
    console.info("Source IP: ", request.event.requestContext.identity.sourceIp);
```